README.md

```diff
@@ -26,6 +26,16 @@ Before you get started:
 - `make install` to deploy universal CRDs
 - `make run` to run the Operator with local go environment
 
+**Notes:**
+1. If `go get ...` is not functioning, an alternative is to:
+  * `cd $GOPATH`
+  * `mkdir -p src/github.com/kudobuilder`
+  * `cd src/github.com/kudobuilder`
+  * `git clone git@github.com:kudobuilder/kudo.git`
+2. **Before** `make install` you will need to have:
+  * minikube running (some of the tests run against it)
+  * `~/.git-credentials` must exist with git credentials.   If you are two-factor auth you will need a create a [personal access token](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line)
```

Comment:
"If you are using" and remove extra spaces.

Reply:
haha... I give... I will stick to 1 space... :)